delivery_deliverea/models/deliverea_distribution_center.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 FactorLibre - Jorge Martínez <jorge.martinez@factorlibre.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class DelivereaDistributionCenter(models.Model):
+    _name = "deliverea.distribution.center"
```

Comment:
It may make sense to use the res.partner model, but a new field would have to be created to identify it from the other distribution centers.

It would be the uuid field.

Reply:
Good afternoon @pedrobaeza 

Yes I have not yet made all the changes, because I have the doubt if the distribution centers model should disappear and add the new field in res.partner or leave that model.

I think that if it could be changed, but creating the field would be mandatory, because you have to differentiate them by id with the new field.

What is your opinion?

Thanks in advance.

Reply:
Is this populated manually or automatically? If automatically, then it's reasonable to keep it isolated in this model.

Reply:
Is automatic, there are a button in the delivery.carrier model to import all the distribution centers.

Reply:
OK, then let's not pollute the partner table for that. It's OK this way :)